src/test_application/activescenes/scenarios_physicstest.cpp

```diff
@@ -498,7 +381,7 @@ void PhysicsTest::setup_renderer_gl(CommonSceneRendererGL& rRenderer, CommonTest
         auto &rControls = rRenderer.get<PhysicsTestControls>();
 
         // Throw a sphere when the throw button is pressed
-        if (rControls.m_camCtrl.m_controls.button_triggered(rControls.m_btnThrow))
+        if (rControls.m_camCtrl.m_controls.button_held(rControls.m_btnThrow))
```

Comment:
Accidentally left the auto-fire enabled. It's more fun though, so I'm keeping it in here.